### PR TITLE
small js docs update

### PR DESCRIPTION
**Pull request by mschultheiss83**
just js docs fixes

### CODE REVIEW

File: src/prototype_creep_harvest.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ Creep.prototype.spawnCarry = function() {
     resourceAtPosition += _.sum(container.store);
   }
 
-  if (resourceAtPosition > parts.carryParts.carry * CARRY_CAPACITY || (this.memory.routing.type === 'commodity' && this.store.getFreeCapacity === 0)) {
+  if (resourceAtPosition > parts.carryParts.carry * CARRY_CAPACITY || (this.memory.routing.type === 'commodity' && this.store.getFreeCapacity() === 0)) {
```
Comment (mschultheiss83) [inline src/prototype_creep_harvest.js:32]: this should be a function call, yes ?
changed `this.store.getFreeCapacity` to `this.store.getFreeCapacity()`

**Comment by mschultheiss83** [comment]
in general should'n be there a updateSettings-Object 
with a keys in roles[role].settings

File: src/prototype_creep_harvest.js (hunk 1 above)
Comment (TooAngel) [reply src/prototype_creep_harvest.js:32]: yes, correct, thx, good catch

**Comment by TooAngel** [comment]
> in general should'n be there a updateSettings-Object
> with a keys in roles[role].settings

I think there was one. I'm not sure if we need it - I was thinking about redoing the creeps config logic.

**Comment by mschultheiss83** [comment]
will merge next week

**Comment by XenoAmess** [comment]
> will merge next week

@mschultheiss83 
Little longer than expected bro...
Please do it.

**Comment by mschultheiss83** [comment]
@XenoAmess can u resolve?
Conflicting files
src/role_carry.js

**Comment by mschultheiss83** [comment]
Just saw I need it needs auto merge

**Review by XenoAmess — APPROVED** [review]

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**